internal/plan/plan.go

```diff
@@ -312,16 +319,21 @@ func ReadLayersDir(dirname string) ([]*Layer, error) {
 		}
 		layers = append(layers, layer)
 	}
-	return layers, nil
+	return CombineLayers(layers...)
```

Comment:
Similarly, it feels like this should return a *Layer, not a *Plan. This is the same operation we'll get when combining the top dynamic layer with the new dynamic layer provided over the wire. So combining feels like a general operation that results a new, combined, layer.

Reply:
Ditto -- fixed.